README.md

```diff
@@ -1,5 +1,87 @@
 # vc-di-ecdsa-test-suite
 
-Interoperability test suite for ECDSA Data Integrity cryptosuites.
+> Test Suite for [ECDSA](https://www.w3.org/TR/vc-di-ecdsa/)
+  [Data Integrity](https://w3c.github.io/vc-data-integrity/) Cryptosuites.
 
-[Data Integrity Spec](https://w3c.github.io/vc-data-integrity/)
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Usage](#usage)
+- [Implementation](#implementation)
+
+## Background
+Provides interoperability tests for verifiable credential processors
+(issuers/verifiers) that support [ECDSA](https://www.w3.org/TR/vc-di-ecdsa/)
+[Data Integrity](https://w3c.github.io/vc-data-integrity/) cryptosuites.
+
+## Install
+
+```js
+npm i
+```
+
+## Usage
+
+```
+npm test
+```
+
+## Implementation
+
+You will need an issuer and verifier that are compatible with [VC API](https://w3c-ccg.github.io/vc-api/)
+and are capable of handling issuance and verification of Verifiable Credentials
+with `DataIntegrityProof` proof type using the `ecdsa-2019` cryptosuite.
+
+To add your implementation to this test suite, you will need to add 2 endpoints
+to your implementation manifest.
+- A credentials issuer endpoint (/credentials/issue) in the `issuers` property.
+- A credentials verifier endpoint (/credentials/verify) in the `verifiers` property.
+
+All endpoints will need the tag `ecdsa-2019`.
+
+A simplified manifest would look like this:
+
+```js
+{
+  "name": "My Company",
+  "implementation": "My implementation",
+  "issuers": [{
+    "id": "",
+    "endpoint": "https://issuer.mycompany.com/credentials/issue",
+    "method": "POST",
+    "tags": ["ecdsa-2019"]
+  }],
+  "verifiers": [{
+    "id": "",
+    "endpoint": "https://verifier.mycompany.com/credentials/verify",
+    "method": "POST",
+    "tags": ["ecdsa-2019"]
+  }]
+}
+```
+
+The example above represents an unauthenticated endpoint. You may add zcap or
+oauth authentication to your endpoints. You can find an example in the
+[README here](https://github.com/w3c-ccg/vc-api-test-suite-implementations#adding-a-new-implementation).
+
+To run the tests, some implementations may require client secrets that can be
+passed as env variables to the test script. To see which ones require client
+secrets, please check the implementation manifest within the
+[vc-api-test-suite-implementations](https://github.com/w3c-ccg/vc-api-test-suite-implementations/tree/main/implementations) library.
+
+### Docker Integration (TODO)
+
+We are presently working on implementing a new feature that will enable the
+utilization of Docker images (using the VC API mentioned above) instead of
+live endpoints.
+
+## License
```

Comment:
I'm not sure we need the license stuff here @msporny is that correct?

Reply:
Yes, we need the license on here because we're transferring this to W3C.